sympy/physics/control/lti.py

```diff
@@ -1300,6 +1300,33 @@ def to_expr(self):
             return Pow(self.den, -1, evaluate=False)
 
 
+class PIDController:
+    def __init__(self, KP, KI, KD, s):
```

Comment:
I understand that `s` is the conventional symbol for continuous time systems but use var as the variable name here

Reply:
sure sir